maze_solver/maze.py

```diff
@@ -63,10 +92,12 @@ def is_traversable(self, x, y):
         Accepts coordinates and return if tile can
         be traversed or not.
         """
-        # TODO
```

Comment:
Para asa ni nga todo? :) and unsay kulang here hihi

Reply:
gi tang tang nani nga code. :)